ansible/configs/kni-osp/files/clouds.yaml.j2

```diff
@@ -0,0 +1,12 @@
+clouds:
+  {{ osp_project_name }}:
+    auth:
+      auth_url: "{{ osp_auth_url }}"
+      username: "{{ guid }}-user"
```

Comment:
This should be changed to use the new var available - `osp_auth_username_member`